.github/workflows/gha.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       # use org's Private Access Token to log in to GitHub Container Registry
       CONTAINER_REG_USER:   ${{ secrets.GH_CR_USER }}
       CONTAINER_REG_PASS:   ${{ secrets.GH_CR_PAT }}
+      FORCE_IMAGE_ACTION:   ${{ secrets.FORCE_IMAGE_ACTION }}
```

Comment:
What will happen, if variable is not defined?

Reply:
It's just not set, which should be fine. But, I just remembered this can't be used on PR (secrets are not available), so it will work only on master branch (on pushes)